### PR TITLE
chore: hide secretary stuff behind digital_agenda flag

### PR DESCRIPTION
**Pull request by kobemertens**
https://kanselarij.atlassian.net/jira/software/c/projects/KAS/boards/1?assignee=63c552d46178fcc941d857c9&selectedIssue=KAS-4318

I only implemented the changes mentioned by Sven in the comment.

### CODE REVIEW

**Comment by kobemertens** [comment]
Sorry for the autoformatter changes

File: app/controllers/agendas.js [hunk 1]
```diff
@@ -196,38 +225,48 @@ export default class AgendasController extends Controller {
...
+      decisionResultCode,
+      ...(this.enableDigitalAgenda
+        ? {
+            secretary: meetingSecretary
+              ? meetingSecretary
+              : currentApplicationSecretary,
+          }
+        : {}),
+      // no subcase. Minutes approval aren't part of a (sub)case
+    });
```
Comment (tdn) [inline app/controllers/agendas.js:227]: Cleaning this up a little bit so the two awaits don't happen when the digital agenda flag is off

File: app/components/meeting/edit-meeting-modal.hbs [hunk 2]
```diff
@@ -115,16 +115,18 @@
...
+          <Auk::Label>
+            {{t "secretary-for-meeting"}}
+          </Auk::Label>
+          <Mandatees::MandateeSelector
+            @onSelectMandatee={{fn (mut this.secretary)}}
+            @selectedMandatees={{this.secretary}}
+            @visibleRoles={{this.visibleRoles}}
+          />
+        </AuFormRow>
+      {{/if}}
```
Comment (tdn) [inline app/components/meeting/edit-meeting-modal.hbs:129]: This hides the component, but the `js` file had numerous references to the secretary as well. Adding the flag there as well for consistency

**Review by tdn — APPROVED** [review]
made some small changes. Merging this as is now.

@SchrodingersCat00 try to loosen the line length in your formatter a bit. These autoformatting changes are a real pain to review and could lead to extra conflicts. If it contributed to readability I wouldn't mind, but in a lot of these cases it just makes things harder to read...

**Pull request merged**